docs/upgrade.md

```diff
@@ -14,5 +14,202 @@ Description: Harvester provides two ways to upgrade. Users can either upgrade us
 !!!note
     Upgrade is not supported from previous versions to v0.3.0 version.
 
-    - A manual upgrade process starting with v0.3.0 is being investigated. Harvester will inform the community once this process is in place.
     - One-click upgrade will be supported starting with the v1.0.0 release.
+
+
+## Manual Upgrade
```

Comment:
should we specify if the Manual Upgrade only supports from `v0.3.0` to `v1.0.0`?
e.g., # Manual Upgrade(From v0.3.0 to v1.0.0)

Reply:
Fixed in 
https://github.com/harvester/docs/pull/67/commits/85589ed563a4486a27409f853207f2af355848b3#diff-9cb3d19a7e30c718742ca50bf7a18ed9a3d057d82eb76fa410358b5b60900e12